src/include/daos/checksum.h

```diff
@@ -96,7 +96,7 @@ struct dcs_iod_csums {
 };
 
 /** Single value layout info for checksum */
-struct dcs_singv_layout {
+struct dcs_layout {
```

Comment:
I'm a little concerned by the prefix, because dsc_ is for the server side client API, dcs_ is quite similar. also the comment is still up-to-date, I mean, is it still for single value?

Reply:
yes, it is for single value. actually the renaming is to follow one of your suggestion of previous patch.

Reply:
ah I forgot.. sorry